### PR TITLE
Improve user guide

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div align="center">
-  <img src="resources/lmdeploy-logo.svg" width="450"/>
+  <img src="docs/en/_static/image/lmdeploy-logo.svg" width="450"/>
 
-[![docs](https://img.shields.io/badge/docs-latest-blue)](https://lmdeploy-zh-cn.readthedocs.io/zh_CN/latest/)
+[![docs](https://img.shields.io/badge/docs-latest-blue)](https://lmdeploy-zh-cn.readthedocs.io/latest/)
```
Comment (RunningLeon) [inline README_zh-CN.md]: may change to `https://lmdeploy.readthedocs.io/zh_CN/latest/`

Comment (lvhan028) [reply README_zh-CN.md]: sure

File: docs/zh_cn/_static/css/readthedocs.css [hunk 2]
```diff
@@ -1,6 +1,62 @@
 .header-logo {
-    background-image: url("../image/lmdeploy-logo.png");
-    background-size: 150px 60px;
+    background-image: url("../image/lmdeploy-logo.svg");
```
Comment (RunningLeon) [inline docs/zh_cn/_static/css/readthedocs.css:2]: There's a wrong url, could fix in this pr:
https://github.com/InternLM/lmdeploy/blob/e7eedc0384a5bf4201d75f7b00f85c2bc60f5a3f/docs/zh_cn/conf.py#L112

File: requirements/docs.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@ m2r==0.2.1
 markdown>=3.4.0
 mistune==0.8.4
 myst-parser
--e git+https://github.com/open-mmlab/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
+-e git+https://github.com/InternLM/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
```
Comment (RunningLeon) [inline requirements/docs.txt:6]: how about `sphinx-book-theme` as used in vllm  https://github.com/vllm-project/vllm/blob/9140561059904f133a3c146f04afe13d1cdd5a6c/docs/requirements-docs.txt#L2 ?

URL
https://sphinx-book-theme.readthedocs.io/en/stable/

File: README_zh-CN.md (hunk 1 above)
Comment (zhulinJulia24) [inline README_zh-CN.md]: 路径不正确，打开错误

File: README_zh-CN.md [hunk 4]
```diff
@@ -39,185 +39,78 @@ ______________________________________________________________________
...
 
-测试配置：LLaMA-7B, NVIDIA A100(80G)
+# 用户教程
 
-TurboMind 的 output token throughput 超过 2000 token/s, 整体比 DeepSpeed 提升约 5% - 15%，比 huggingface transformers 提升 2.3 倍
-在 request throughput 指标上，TurboMind 的效率比 vLLM 高 30%
+请阅读[快速上手](./docs/zh_cn/get_started.md)章节，了解 LMDeploy 的基本用法。
 
-![benchmark](https://github.com/InternLM/lmdeploy/assets/4560679/7775c518-608e-4e5b-be73-7645a444e774)
+为了帮助用户更进一步了解 LMDeploy，我们准备了用户指南和进阶指南，请阅读我们的[文档](https://lmdeploy.readthedocs.io/en/latest/)：
```
Comment (zhulinJulia24) [inline README_zh-CN.md]: 链接使用中文版

File: docs/en/get_started.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+## Quantization
+
+### Weight INT4 Quantization
+
+LMDeploy uses [AWQ](https://arxiv.org/abs/2306.00978) algorithm for model weight quantization
+
+Using the following commands, you can quantize a LLM model into 4bit, and communicate with it with command line:
+
+```shell
+lmdeploy lite calibrate internlm/internlm-chat-7b --work-dir ./internlm-chat-7b-4bit
```
Comment (zhulinJulia24) [inline docs/en/get_started.md:51]: https://github.com/InternLM/lmdeploy/blob/a7c1e6219b17b0241905192842664a8a8ff29891/docs/en/quantization/w4a16.md  和文档里的command的不同少了较多的入参，那些入参会有默认值么？

Comment (lvhan028) [reply docs/en/get_started.md:51]: 是的，都有默认值

File: README_zh-CN.md (hunk 4 above)
Comment (lvhan028) [reply README_zh-CN.md]: 已修复

File: README_zh-CN.md (hunk 1 above)
Comment (lvhan028) [reply README_zh-CN.md]: 已修复

File: docs/zh_cn/_static/css/readthedocs.css (hunk 2 above)
Comment (lvhan028) [reply docs/zh_cn/_static/css/readthedocs.css:2]: fixed

File: README_zh-CN.md [hunk 6]
```diff
@@ -18,10 +18,10 @@
 
 ______________________________________________________________________
 
-## 更新 🎉
+## 最新进展 🎉
 
 - \[2023/12\] Turbomind 支持多模态输入。[Gradio Demo](./examples/vl/README.md)
-- \[2023/11\] Turbomind 支持直接读取 Huggingface 模型。点击[这里](./docs/en/load_hf.md)查看使用方法
+- \[2023/11\] Turbomind 支持直接读取 Huggingface 模型。点击[这里](docs/en/inference/load_hf.md)查看使用方法
```
[the remark this answers is not in the document]
Comment (lvhan028) [reply README_zh-CN.md]: 已修复

**Review by grimoire — APPROVED** [review]

**Review by AllentDan — APPROVED** [review]

**Review by zhulinJulia24 — APPROVED** [review]
lgtm

**Comment by irexyc** [comment]
pycuda安装需要编译，gcc版本过低会安装失败，这个要说明么？

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**